### PR TITLE
Feature/2 - Splash Screen

### PR DESCRIPTION
**Pull request by ghost**
-  Splash Screen files for iPhones from the device matrix; 
- Tags referencing the image files in the index.html;

JIRA Items:

Task 2

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -7,6 +7,12 @@
     <meta name="theme-color" content="#000000" />
     <meta name="apple-mobile-web-app-capable" content="yes">
     <link rel="apple-touch-icon" href="assets/icon.png">
+    <!-- iPhone Xs Max (1242px x 2688px) -->
```
Comment (arctouch-jonathansouza) [inline public/index.html]: No need to add these comments. This information is already explicit on the `<link>` tag declarations.

**Review by arctouch-jonathansouza — DISMISSED** [review]
Just added one comment. LGTM after addressing it! 👍

File: public/index.html (hunk 1 above)
Comment (arctouch-alyssonlopes) [reply public/index.html]: done

**Review by arctouch-jonathansouza — APPROVED** [review]

**Review by arctouch-marceloaquino — APPROVED** [review]
Looks good.

**Pull request merged**